homeassistant/components/mqtt/client.py

```diff
@@ -611,7 +613,7 @@ async def publish_birth_message(birth_message):
                 publish_birth_message(birth_message), self.hass.loop
             )
 
-    def _mqtt_on_message(self, _mqttc, _userdata, msg) -> None:
+    def _mqtt_on_message(self, _mqttc, _userdata, msg: MQTTMessage) -> None:
```

Comment:
The callback method is only partially typed.
`_mqttc` is probably `mqt.Client`, whereas `_userdata` is likely `None` AFAICT.

Reply:
I thought there would be import problems, but that is not the fact. Solved now.